code/index.html

```diff
@@ -1,21 +1,367 @@
+            <div class="wunsch-Bild">
+              <img
+                class="image-wunsch"
+                src="./images/Abendessen.jpg"
+                alt="Bild zNacht"
+              />
+            </div>
+            <div class="wunsch-Text">
+              <p>
+                Gemütliches Nachtessen <br />
+                <strong>100 CHF</strong>
+              </p>
+            </div>
+          </div>
+        </li>
+        <li class="wishlist">
+          <div class="wunsch-container">
+            <div class="wunsch-Bild">
+              <img
+                class="image-wunsch"
+                src="./images/kleineAktivität.jpg"
+                alt="Bild zNacht"
+              />
+            </div>
+            <div class="wunsch-text">
+              <p>
+                Kleine Aktivität<br />
+                <strong>50 CHF</strong>
+              </p>
+            </div>
+          </div>
+        </li>
+        <li class="wishlist">
+          <div class="wunsch-container">
+            <div class="wunsch-Bild">
+              <img
+                class="image-wunsch"
+                src="./images/grosseAktivität.jpg"
+                alt="Bild zNacht"
+              />
+            </div>
+            <div class="wunsch-text">
+              <p>Grosse Aktivität<br /><strong>100 CHF</strong></p>
+            </div>
+          </div>
+        </li>
+        <li class="wishlist">
+          <div class="wunsch-container">
+            <div class="wunsch-Bild">
+              <img
+                class="image-wunsch"
+                src="./images/Feuer.jpg"
+                alt="Bild zNacht"
+              />
+            </div>
+            <div class="wunsch-text">
+              <p>
+                Snacks für gemütlichen Abend am Feuer<br /><strong
+                  >30 CHF</strong
+                >
+              </p>
+            </div>
+          </div>
+        </li>
+        <li class="wishlist">
+          <div class="wunsch-container">
+            <div class="wunsch-Bild">
+              <img
+                class="image-wunsch"
+                src="./images/Bier.jpg"
+                alt="Bild zNacht"
+              />
+            </div>
+            <div class="wunsch-text">
+              <p>Ein kühles Bier<br /><strong>40 CHF</strong></p>
+            </div>
+          </div>
+        </li>
+        <li class="wishlist">
+          <div class="wunsch-container">
+            <div class="wunsch-Bild">
+              <img
+                class="image-wunsch"
+                src="./images/Bier.jpg"
+                alt="Bild zNacht"
+              />
+            </div>
+            <div class="wunsch-text">
+              <p>Ein kühles Bier<br /><strong>40 CHF</strong></p>
+            </div>
+          </div>
+        </li>
+      </ul>
+    </div>
+    <footer>
+      <p class="footer-text">
+        Hier steht irgendein Text, den ihr in den Footer schreiben möchtet.
```

Comment:
To increase readability of the text, I'd recommend using a lighter text color in the footer, e.g. `color: white`